### PR TITLE
Improvements for es.json

### PR DESCRIPTION
**Pull request by Angel3245**
- Grammatical and spelling errors fixed

### CODE REVIEW

**Review by CarlosNunezMX — COMMENTED** [review]
"Mola" is a traduction for the Spanish for Spain but in latam doesn't use it, because i put "Yeahs"

**Review by CarlosNunezMX — CHANGES_REQUESTED** [review]
But i i think on Change "mola" for "Genial" or "Me gusta"

**Comment by DkStar7** [comment]
Sounds great to me, but I would change the part where it says "Mostrar ____ en Perfil" to "Mostrar ____ en tu perfil". Además de cambiar "anuncios" to "noticias" Apart from that, the translation is perfect for me.

**Review by DkStar7 — COMMENTED** [review]
It's a small change. Apart from that, the translation is perfect.

File: src/translations/es.json [hunk 1]
```diff
@@ -51,20 +51,20 @@
...
     "unfriend": "Eliminar amigo",
     "no_friends": "Sin amigos",
-    "no_following": "No sigue a nadie",
-    "no_followers": "No tiene seguidores"
+    "no_following": "No sigues a nadie",
+    "no_followers": "No tienes seguidores"
   },
   "user_settings": {
     "profile_settings": "Preferencias del Perfil",
     "show_country": "Mostrar país en Perfil",
```
[the remark this answers is not in the document]
Comment (CarlosNunezMX) [reply src/translations/es.json:59]: Sounds great

[the remark this answers is not in the document]
Comment (Angel3245) [reply src/translations/es.json]: Okay, i'll change it.

**Comment by Angel3245** [comment]
> But i i think on Change "mola" for "Genial" or "Me gusta"

I wrote "Mola" because it's the official translation for that on Miiverse, but "Genial" or "Me gusta" are also okay.

**Comment by CarlosNunezMX** [comment]
> > But i i think on Change "mola" for "Genial" or "Me gusta"
> 
> I wrote "Mola" because it's the official translation for that on Miiverse, but "Genial" or "Me gusta" are also okay.

Yep or "Me gusta" is a common expression on Latam and Spain.

**Comment by DkStar7** [comment]
You can change "página personal" by "perfil"

**Pull request merged**